pallets/stateful-storage/README.md

```diff
@@ -0,0 +1,61 @@
+# Stateful Storage Pallet
+
+The Stateful Storage pallet provides per-MSA functionality for reading and writing stateful data (where only the latest state is relevant).
+
+## Summary
+
+When a Schema uses `Paginated` or `Itemized`, the payload data is stored in this pallet.
+Data stored is user-centric instead of time-centric as with the Messages Pallet.
+The pallet storage uses [child storage](https://paritytech.github.io/polkadot-sdk/master/frame_support/storage/child/index.html) making direct query access complex.
+Custom RPCs are provided for easy access to data.
+
+### Paginated Data (`PayloadLocation:Paginated`)
+
+Data is stored in multiple pages of `1_024` bytes in size (limit defined by `constants::MaxPaginatedPageSizeBytes`), each containing a single item of the associated schema.
```

Comment:
This might be a little easier to read:
Data is stored in multiple pages, each 1,024 bytes in size (as defined by constants::MaxPaginatedPageSizeBytes). Each page contains a single item of the associated schema.

Reply:
I like it.
```suggestion
Data is stored in multiple pages, each `1_024` bytes in size (as defined by `constants::MaxPaginatedPageSizeBytes`).
Each page contains a single item of the associated schema.
```